packages/flutter/test/material/checkbox_test.dart

```diff
@@ -2331,3 +2360,31 @@ class _SelectedGrabMouseCursor extends MaterialStateMouseCursor {
   @override
   String get debugDescription => '_SelectedGrabMouseCursor()';
 }
+
+Widget buildAdaptiveCheckbox({
+  required TargetPlatform platform,
+  RoundedRectangleBorder shape = const RoundedRectangleBorder(),
+  BorderSide side = const BorderSide(),
+  bool value = true,
+  bool enabled = true,
+}) {
+  final Widget adaptiveCheckbox = Checkbox.adaptive(
```

Comment:
Why not instantiate this inline?

Reply:
I will have to write more tests, and for every test, I would need to instantiate inline (with possibly different parameters depending on the test). Would it be best to instantiate inline in this case, or just have one definition I can call with different parameters across the test suite?